### PR TITLE
Refactored AppView list fetching

### LINKED ISSUE #73: Add UserName to metadata

### PR DESCRIPTION
**Pull request by Nephim**

### CODE REVIEW

File: src/UICrafter.Mobile/Components/Pages/AuthPage.razor
Comment (Aagaard00) [inline src/UICrafter.Mobile/Components/Pages/AuthPage.razor:1]: Delete this file if its not used

File: src/UICrafter/Components/App.razor [hunk 1]
```diff
@@ -18,7 +18,7 @@
 <body>
     @if (HasTheme)
     {
-        <Routes @rendermode="InteractiveAuto" Theme="theme" />
+        <Routes @rendermode="InteractiveServer" Theme="theme" />
```
Comment (Aagaard00) [inline src/UICrafter/Components/App.razor]: Change back to InteractiveAuto

**Review by Aagaard00 — CHANGES_REQUESTED** [review]
After requested changes -> LGTM

**Review by Gormiee — APPROVED** [review]
LGTM

**Pull request merged**